admin/assets/javascripts/discourse/templates/update-index.hbs

```diff
@@ -1,4 +1,21 @@
-<h1>{{i18n "admin.docker.update_title"}}</h1>
+<div class="updates-heading">
+  <h3>{{i18n "admin.docker.update_title"}}</h3>
+  {{#unless this.outdated}}
+    <button
```

Comment:
Why not use `DButton` for this? This one has currently got slightly different styling to the Update buttons in the list